std-bits/table/src/main/java/org/enso/table/data/table/Column.java

```diff
@@ -216,4 +220,30 @@ Column mask(IndexMapper indexMapper) {
   public List<?> asList() {
     return new StorageListView(this.getStorage());
   }
+
+  private static ColumnStorage<?> fixStorage(ColumnStorage<?> inputStorage) {
+    if (Proxy.isProxyClass(inputStorage.getClass())) {
+      var proxyType = inputStorage.getType();
+      var type = StorageType.fromTypeCharAndSize(proxyType.typeChar(), proxyType.size());
+      if (type == IntegerType.INT_64) {
+        var addressData = inputStorage.rawData();
+        var addressNothings = inputStorage.rawNothings();
```

Comment:
- ideally this would be just a single `rawData` `Buffer`
- that means we need to differentiate between _non-null and with-nulls_ `StorageType`
   - should `StorageType` be expanded to have `boolean hasNothings()`, @jdunkerley?
- then we could probably reuse @hubertp's [ByteBufferDirect](https://github.com/enso-org/enso/blob/e9876293b835cac2878858415901b95c7fe53bf2/engine/runtime-language-arrow/src/main/java/org/enso/interpreter/arrow/runtime/ByteBufferDirect.java#L19) somehow
   - looks like it is capable to hold both sets of info (values & nullness) in the same `Buffer`